tslint.json

```diff
@@ -1,5 +1,26 @@
 {
+  "rulesDirectory": [
+    "node_modules/codelyzer"
+  ],
   "rules": {
+    "directive-selector-name": [true, "camelCase"],
+    "component-selector-name": [true, "kebab-case"],
+    "directive-selector-type": [true, "attribute"],
+    "component-selector-type": [true, "element"],
+    "directive-selector-prefix": [true, "sg"],
```

Comment:
Is this a thing where you can configure codelyzer to use a custom prefix of your choosing?

If so, that's cool, but we should update the value to 'rio', which is this project's prefix.